README.rst

```diff
@@ -236,7 +235,7 @@ Visual Studio Code
     "python.formatting.blackPath": "<install_location_from_step_2>",
     "python.formatting.blackArgs": ["--diff"],
 
-Extra argument can be passed to ``darker`` along side, but ``--diff`` is necessary.
+You can pass additional arguments to ``darker`` in the ``blackArgs`` option (e.g. ``["--diff", "--isort"]``), but make sure at least ``--diff`` is included.
```

Comment:
Actually, could we keep the README file a bit narrower for the benefit of people reading it unrendered? Sorry for not noticing this in my original suggestion.

```suggestion
You can pass additional arguments to ``darker`` in the ``blackArgs`` option
(e.g. ``["--diff", "--isort"]``), but make sure at least ``--diff`` is included.
```

Reply:
Oh, yeah did not realize either.